website/src/components/GridPattern.tsx

```diff
@@ -0,0 +1,24 @@
+import { useId } from 'react'
+
+export function GridPattern(
```

Comment:
**suggestion**: I thought it was a grid component. I'd rename it in `BackgroundGridPattern` to make its use clearer